src/Shared/HttpSys/Extensions.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.HttpSys.Internal
```

Comment:
I'm not really happy with this namespace, as it used now in IIS too. Is there a better name / place for this extension?

Reply:
It's fine. IIS depends on HttpSys for all transport level functionality so consuming an HttpSys namespace is expected.